### ISSUE TITLE
Indicate source for each entry

### ISSUE THREAD

**Issue by krlmlr** [anthonynorth/roxyglobals#3]
Should each entry in `globals.R` indicate the file(s) where this entry has been added from in a comment?

```r
utils::globalVariables(c(
  "end_time", # summarise_responses
  "start_time", # summarise_responses
  "station_name", # summarise_responses
  "station_type", # summarise_responses
  NULL
))
```

**Comment by anthonynorth** [comment]
I think that's reasonable. I'm currently removing duplicates prior to writing to globals.R, so it probably makes sense to just write out all variables as they are declared.

**Comment by anthonynorth** [comment]
Implemented in [v0.2.1](https://github.com/anthonynorth/roxyglobals/releases/tag/v0.2.1)
